templates/scenarios/ts/notification-http-timer-trigger/src/httpTrigger.ts

```diff
@@ -21,61 +21,87 @@ const httpTrigger: AzureFunction = async function (
 ): Promise<void> {
   // By default this function will iterate all the installation points and send an Adaptive Card
   // to every installation.
-  for (const target of await notificationApp.notification.installations()) {
-    await target.sendAdaptiveCard(
-      AdaptiveCards.declare<CardData>(notificationTemplate).render({
-        title: "New Event Occurred!",
-        appName: "Contoso App Notification",
-        description: `This is a sample http-triggered notification to ${target.type}`,
-        notificationUrl: "https://aka.ms/teamsfx-notification-new",
-      })
+  const pageSize = 100;
+  let continuousToken: string | undefined = undefined;
```

Comment:
contin***uation***Token?

Reply:
It seems some files use `continuationToken` while some use `continuousToken`.

Reply:
nice catch! will consolidate it.